docs/how-to/monitor-your-backups.md

```diff
@@ -562,6 +563,58 @@ Heartbeat Retry = 360          # = 10 minutes
 Resend Notification every X times = 1
 ```
 
+## zabbix hook
+
+<span class="minilink minilink-addedin">New in version 1.8.15</span>
+[zabbix](https://www.zabbix.com/) is an open-source monitoring tool used for tracking and managing the performance and availability of networks, servers, and applications in real-time.
+
+This hook does not do any notifications on it's own. Instead, it relies on
+your zabbix instance to notify and perform escalations based on the zabbix
+configuration. The `states` list will choose which states to trigger on. 
+Each state can have its own custom values configured. These values are 
+populated in the item data in zabbix. If none are provided, will use the
```

Comment:
I don't know what "in the item data" means, although this might just be Zabbix terminology. Maybe it's referring to a part of the API call's data?

Reply:
My wording may not be the best. What I am trying to say is that the values entered in the states are what will be entered into zabbix's item.

In the below example, zabbix will populate "STARTED" into the value of the item on Zabbix.

```yaml
    start:
        value: "STARTED"
    finish:
        value: "OK"
    fail:
        value: "ERROR"
```

Reply:
Gotcha. Then maybe something like this would convey that?

> And a value is used to populate the data of the configured Zabbix item.

Or if you wanted to get even more verbose and specific:

> And a value is used to populate the item corresponding to the configured Zabbix itemid.

Of course, feel free to wordsmith.

Reply:
Yup, looks good to me!